components/feature/media/src/main/java/mozilla/components/feature/media/ext/MediaState.kt

```diff
@@ -28,6 +28,28 @@ fun MediaState.getPlayingMediaIdsForTab(tabId: String?): List<String> {
         ?: emptyList()
 }
 
+/**
+ * Finds any fullscreen video status and orientation of fullscreen media.
+ */
+fun MediaState.findFullscreenMediaOrientation(): MediaState.FullscreenOrientation? {
```

Comment:
nit: This can be `internal` and is not expected to be called by users of the library, I think?